--- origsrc/babl-0.1.0/extensions/Makefile.am	2009-09-22 12:10:03.809726900 -0500
+++ src/babl-0.1.0/extensions/Makefile.am	2009-09-22 12:24:50.122927400 -0500
@@ -12,7 +12,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/babl		\
 	-I$(top_srcdir)/extensions
 
-extdir = $(DESTDIR)$(libdir)/babl-@BABL_API_VERSION@
+extdir = $(libdir)/babl-@BABL_API_VERSION@
 ext_LTLIBRARIES = \
 	CIE-Lab.la      \
 	gegl-fixups.la  \
